pages/core-unit/[code]/index.tsx

```diff
@@ -1,11 +1,25 @@
-import { NextPage } from 'next';
+import { NextPage, GetServerSideProps, InferGetServerSidePropsType, GetServerSidePropsContext } from 'next';
 import React from 'react';
+
 import CuAboutContainer from '../../../src/stories/containers/cu-about/cu-about-container';
+import { CuAbout, fetchCoreUnitByCode } from '../../../src/stories/containers/cu-about/cu-about.api';
 
-const CoreUnitAboutPage:NextPage = () => {
+const CoreUnitAboutPage: NextPage = ({ code, cuAbout, contributors }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   return (
-    <CuAboutContainer />
+    <CuAboutContainer code={code} cuAbout={cuAbout as CuAbout} contributors={contributors} />
   );
 };
-
 export default CoreUnitAboutPage;
+export const getServerSideProps: GetServerSideProps = async(context: GetServerSidePropsContext) => {
+  const { query } = context;
+  const code = query.code as string;
+  const cuAbout = await fetchCoreUnitByCode(code);
```

Comment:
A 404 error should be thrown when there are any CoreUnit with the given code. Otherwise an empty page will be rendered

See https://nextjs.org/blog/next-10#notfound-support

Reply:
you are right , i will add it